cognite/neat/workflows/steps/lib/rules_importer.py

```diff
@@ -0,0 +1,414 @@
+import json
+import logging
+from datetime import datetime
+from pathlib import Path
+from typing import ClassVar
+
+import yaml
+from rdflib import Namespace
+
+from cognite.neat.rules.models import Class, Metadata, Property, TransformationRules
+from cognite.neat.workflows.model import FlowMessage
+from cognite.neat.workflows.steps.data_contracts import RulesData
+from cognite.neat.workflows.steps.step_model import Configurable, Step
+
+CATEGORY = __name__.split(".")[-1].replace("_", " ").title()
+
+__all__ = ["OpenApiToRules", "ArbitraryJsonYamlToRules"]
+
+
+class OpenApiToRules(Step):
+    """The step extracts schema from OpenApi/Swagger specification and generates NEAT transformation rules object."""
+
+    description = "The step extracts schema from OpenAPI specification and generates NEAT transformation rules object. \
+    The rules object can be serialized to excel file or used directly in other steps."
+    category = CATEGORY
+    version = "0.1.0-alpha"
+    configurables: ClassVar[list[Configurable]] = [
+        Configurable(
+            name="openapi_spec_file_path",
+            value="workflows/openapi_to_rules/source_data/openapi.json",
+            label="Relative path to the OpenAPI spec file.The file can be in either json or in yaml format.",
+        )
+    ]
+
+    def run(self) -> (FlowMessage, RulesData):
+        openapi_file_path = Path(self.data_store_path) / Path(self.configs["openapi_spec_file_path"])
+        self.processed_classes_counter = 0
+        self.processed_properies_counter = 0
+        self.failed_classes_counter = 0
+        self.failed_properties_counter = 0
+        self.failed_classes: dict[str, str] = {}
+        self.failed_properties: dict[str, str] = {}
+        rules = self.open_api_to_rules(openapi_file_path)
```

Comment:
Why are these class attributes? They are only used in this function, lets just make them regular variables.